mirror/create_mirror_misc.sh

```diff
@@ -8,6 +8,8 @@ fi
 [[ -z ${MIRROR_BUILD_DIR} ]] && export MIRROR_BUILD_DIR=${PWD}
 [[ -z ${MIRROR_OUTPUT_DIR} ]] && export MIRROR_OUTPUT_DIR=${PWD}/mirror-dist
 
+export ANACONDA_VERSION="5.1.0"
```

Comment:
This feels upside down to me. Shouldn't the version definition itself be in the (or a) dependencies definition and used here?